src/Api/API.php

```diff
@@ -3325,6 +3325,17 @@ protected function applyMassiveAction(
         $action = explode(':', $action_key);
         $processor = $action[0];
 
+        if ($action[1] === 'update') {
```

Comment:
I do not know if this endpoint is supposed to be used like this. We should add a comment to explain what is done here.
Here is the payload sent when using update massive action from UI:
![image](https://user-images.githubusercontent.com/33253653/212916459-01d910bf-09c3-486f-8075-97006c482338.png)

As far as I remember, payload on API should be similar.

@AdrienClairembault Do you remember how it is supposed to work?

Reply:
Either some update broke the API side or this specific call never worked. hard to tell without checking back the original changes (#9205).

The proposed solution seems OK as it keep the request input simple.

Reply:
Problem with current change is that it may break existing calls that are using expected payloads. I do not know if such calls exists, and I am not sure of what is the expected payload. As far as I remember, the `getMassiveActionParameters` method should tell what are the expected parameters.